apps/cookbook/src/app/examples/list/components/load-on-demand/list-load-on-demand-example.component.ts

```diff
@@ -1,13 +1,27 @@
 import { Component } from '@angular/core';
-import { of } from 'rxjs';
 
 import { BaseListComponent } from '../base-list.component';
 import { LoadOnDemandEvent } from '@kirbydesign/designsystem';
 
-@Component({
+export const config = {
```

Comment:
Consider aligning with other showcase examples:
Export the config.template as an instance member and get the template from the example instance in the markup